docs/pages/database-access/guides/gui-clients.mdx

```diff
@@ -62,6 +62,14 @@ In the "SSL" tab, set "SSL Mode" to `Verify-Full`:
 
 Click "Save", and pgAdmin should immediately connect.
 
+<Admonition type="note">
+  On a Windows client, use terminal to export a PG Service File env variable:
+
+ `$ setx PGSERVICEFILE <path_to_.pg_service.conf_file>`
```

Comment:
This should probably use the new `code` snippet type. Can't insert example here because Github tries to render it as markdown :) Take a look in other mdx files, e.g. CLI reference.